src/registrar/tests/test_models_domain.py

```diff
@@ -578,21 +697,91 @@ def test_user_removes_some_nameservers(self):
                 to the registry
             And `domain.is_active` returns True
         """
-        raise
 
-    @skip("not implemented yet")
+        # Mock is set to return 3 nameservers on infodomain
+        self.extendedValues = True
```

Comment:
**suggestion**: This feels a little magical because of the way it interacts with _getattrshelper. Can we give it a more specific name or use a separate mock so we don't have to change it with this instance attribute?